README.md

```diff
@@ -174,3 +113,70 @@ That is, the voltage levels between two voltage levels which are connected in th
 <p align="center">
 <img src=".github/partial_diagram_example_1_25.svg?sanitize=true" alt="Diagram IEEE30 partial network VL1-VL25" width="65%"/>
 </p>
+
+
+## PowSyBl vs PowSyBl Single Line Diagram
+
+PowSyBl Single Line Diagram is a component build on top of the `Network` model available in the PowSyBl Core repository responsible 
+for generating a [single line diagram](https://en.wikipedia.org/wiki/One-line_diagram).
+
+The main features are:
+ - Node/Breaker and bus/breaker topology.
+ - [SVG](https://fr.wikipedia.org/wiki/Scalable_Vector_Graphics) diagram to be used in various front-end technologies.
+ - Voltage-level, substation and zone diagrams.
+ - Highly customizable rendering using equipment component libraries, CSS and configurable labels (position and content).
+ - Multiple layout modes: fully automatic, semi-automatic (using relative positions for busbar sections and feeders), CGMES DL.
+
+![Diagram demo](.github/diagram-demo.svg)
+*The example above corresponds to a CGMES file from the ENTSO-E sample files.*
+*A guide to generate this diagram is available [here](https://www.powsybl.org/pages/documentation/developer/api_guide/svg-writing.html).*
```

Comment:
I suggest to just remove the code examples (both for NAD and SLD) from the README and simply link to the website:
* To me, the SLD part in the README is redundant with the tutorial. I was thinking of removing it.
* As to the NAD part, I suggest to move it to the website and create a small tutorial with it.
It would look cleaner.

And why not give the links to the tutorials and the api guide in the getting started section ?

What do you think ?